authentication/managing_cloud_provider_credentials/cco-mode-passthrough.adoc

```diff
@@ -0,0 +1,99 @@
+[id="cco-mode-passthrough"]
+= Using passthrough mode
+include::modules/common-attributes.adoc[]
+:context: cco-mode-passthrough
+toc::[]
+
+Passthrough mode is supported for Amazon Web Services (AWS), Microsoft Azure, Google Cloud Platform (GCP), {rh-openstack-first}, {rh-virtualization-first}, and VMware vSphere.
+
+In passthrough mode, the Cloud Credential Operator (CCO) passes the provided cloud credential to the components that request cloud credentials. The credential must have permissions to perform the installation and complete the operations that are required by components in the cluster, but does not need to be able to create new credentials. The CCO does not attempt to create additional limited-scoped credentials in passthrough mode.
+
+[id="passthrough-mode-permissions"]
+== Passthrough mode permissions requirements
+When using the CCO in passthrough mode, ensure that the credential you provide meets the requirements of the cloud on which you are running or installing {product-title}. If the provided credentials the CCO passes to a component that creates a `CredentialsRequest` CR are not sufficient, that component will report an error when it tries to call an API that it does not have permissions for.
```

Comment:
Is this done automatically? As suggested in _[Default behavior](https://github.com/openshift/openshift-docs/pull/29545/files#diff-76786602ed2bd71315fdf4b992f73b242cf54a482cf18d893838489034b7405eR75)_?

Reply:
Tracking in [OSDOCS-1817](https://issues.redhat.com/browse/OSDOCS-1817)